ProgrammingFundamentalsFinalExamPractise/string-manipulator.js

```diff
@@ -0,0 +1,144 @@
+/**
```

Comment:
This file looks good to me. Now refactor others in same way.

Reply:
Ok